kolibri/core/content/fixtures/longdescriptions_content_data.json

```diff
@@ -91,6 +91,10 @@
     {
       "id": "222455c2cc484298b1501a13e1c7eb5c",
       "tag_name": "tag_3"
+    },
+    {
+      "id": "0c20e2eb254b4070a713da63380ff0a3",
+      "tag_name": "velocidad de reacciones químicas"
```

Comment:
This looks like this is still ASCII rather than having any UTF-8 characters?

Reply:
yep, this test is to check 32 chars are cut to 30, just that. I used a real case from channel `07cd1633691b4473b6fda08caf826253` just for the _pleasure_ of using real data

Reply:
Added a new test _test_utf_test_keeps_its_length_ to check utf-8 strings are not byte encoded